setup/config/webpack.config.js

```diff
@@ -43,6 +43,7 @@ const config = {
       crypto: require.resolve('crypto-browserify'),
       stream: require.resolve('stream-browserify'),
       path: require.resolve('path-browserify'),
+      querystring: require.resolve("querystring-es3")
```

Comment:
I had to include it in webpack alias because webpack could not bundle on jenkins this was the erro it throw

`
ERROR in ./node_modules/@liskhq/lisk-cryptography/dist-node/encrypt.js 5:20-42
Module not found: Error: Can't resolve 'querystring' in '/home/lisk/workspace/lisk-desktop_PR-4518/node_modules/@liskhq/lisk-cryptography/dist-node'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "querystring": require.resolve("querystring-es3") }'
	- install 'querystring-es3'
`